lib/src/main/java/io/ably/lib/types/Message.java

```diff
@@ -112,7 +112,7 @@ Message readMsgpack(MessageUnpacker unpacker) throws IOException {
 			if(fieldName.equals("name")) {
 				name = unpacker.unpackString();
 			} else if (fieldName == "extras") {
```

Comment:
Our static analysis engine detects that your pull request includes a change from a Comparison of String objects using `==` or `!=`. According to [this description](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#es-comparison-of-string-objects-using-or-es-comparing-strings-with-eq), unless both strings are either constants in a source file, or have been interned using the `String.intern()` method, the same string value may be represented by two different String objects. Consider using the `equals(Object)` method instead.